### PR TITLE
WD : Initial Setup and Added static pages

### PR DESCRIPTION
**Pull request by devananth**
### Added Static Pages

   - Made an initial setup for the video library project.
   - Created some basic **components**
   - Added *Navbar, Drawer* components
   - Created static pages and added routes
   - Made *Login, Singup* pages, ( functionalities will be added later )

**Screenshots:**

image

image

image

image

image

### CODE REVIEW

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean I liked that you have made the form button component and form Container  Components

File: src/components/drawer/Drawer.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    text: "History",
+  },
+];
+
+const DrawerItem = ({ id, icon, text, path }) => {
+  return (
+    <li className="drawer__options">
+      <NavLink to={path}>
+        <span className="drawer__icon">
+          <i className={icon} aria-hidden="true"></i>
```
Comment (ajit-sharmaGH) [inline src/components/drawer/Drawer.jsx:43]: this is really good, using icons in destructured format and others as well

File: src/components/drawer/drawer.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    .drawer__options .drawer__text{
+        font-size: 1rem;
+        display: inline-block;
+    }
+
+    .drawer__options a{
+        display : flex;
+        flex-direction: column;
+        gap : .5rem;
```
Comment (ajit-sharmaGH) [inline src/components/drawer/drawer.css]: `0.5rem` for better readability

File: src/pages/Authentication/Login/login.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  const loginFocusHandler = (event) => {
+    focusHandler(event, loginFormErrorDispatch);
+  };
+
+  return (
+    <>
+      <Navbar />
+      <section
+        className="d-flex xy-center"
+        style={{ minHeight: "calc(100vh - 4rem)" }}
```
Comment (ajit-sharmaGH) [inline src/pages/Authentication/Login/login.jsx]: you can create an utility className like `body-min-height`  for this, avoid inline style !

**Review by ajit-sharmaGH — APPROVED** [review]
Your code structures are really good, clean and readable, you can go ahead,
great use of utility classes !

**Pull request merged**